### PR TITLE
Added JKL Hotkeys

### LINKED ISSUE #334: Bind jkl to video controls like YouTube (back 10s, pause, and forward 10s respectively)
title

### PR DESCRIPTION
**Pull request by ChueyB**
J: Backwards 10
K: Play/Pause
L: Forward 10

Closes #334

### CODE REVIEW

File: src/renderer/Layout.tsx [hunk 1]
```diff
@@ -128,6 +129,21 @@ const Layout: React.FC<IProps> = (props: IProps) => {
       fill: true,
       inactivityTimeout: 0,
       playbackRates: [0.25, 0.5, 1, 1.5, 2],
+      plugins: {
+        hotkeys: {
+          seekStep: 10,
+          enableModifiersForNumbers: false,
+          forwardKey(event: { which: number }) {
+            return event.which === 76 || event.which === 39;
```
Comment (aza547) [inline src/renderer/Layout.tsx:137]: I think the 74, 75 and 75 are just ASCII here. What's the 32, 37, 39 etc? 

Didn't see a reference for what this actually is on the videojs-hotkeys page, probably could do with a comment.

**Comment by ChueyB** [comment]
Added a comment to the file. These are [keyCode events](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode). 
keyCode

**Comment by ChueyB** [comment]
Quick change of mind. Removed the comments and refactored to make it more intuitive and the result should not necessitate the need for comments

**Review by aza547 — APPROVED** [review]
Looks great!

**Pull request merged**